.github/workflows/main.yml

```diff
@@ -32,10 +33,18 @@ on:
         description: 'Build and publish documentation '     
         required: false
         default: 'false'
+      docLatest:
+        description: 'Build and publish documentation - Latest'     
+        required: false
+        default: 'false'
       dockerPush:
         description: 'Docker push'     
         required: false
-        default: 'false'       
+        default: 'false'      
```

Comment:
Is it normal that there are changes to the actual workflow along with the documentation?